pages/account/manage/events.js

```diff
@@ -71,7 +71,7 @@ export default function ManageEvents({ events }) {
         )}
 
         <Navigation />
-        <Button href="/account/manage/event">
+        <Button href="/account/manage/event" style={{"margin-bottom": "12px"}}>
```

Comment:
Please use tailwind classes

Reply:
Hi eddie!

for that line of code i didn't use tailwind classes because when i used it it will override the default classes from `Button` component, and the button will look like this
![image](https://github.com/EddieHubCommunity/BioDrop/assets/53510222/252a3320-7be5-43d1-9bdc-2829f8ea0a6f)

that's why i'm using the style props

Reply:
Should i add a new props? Like whether the button used for manage pages or not. 
Or maybe just add the margin to the button component?

Reply:
Thank you for making the changes. I think in most places it looks ok, on the map page the button is a bit off but I don't think a blocker

<img width="1027" alt="Screenshot 2023-09-01 at 10 18 52" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/d13c4288-0e54-4e20-929e-bc14decd481f">